.circleci/config.yml

```diff
@@ -355,6 +355,25 @@ jobs:
           command: |
             pnpm --dir integration-testing/bsd test
 
+  test-integration-testing-election-manager:
+    docker:
+      - image: circleci/python:3.7.7-node-browsers
```

Comment:
What version of node is this using?

Reply:
Answering my own question: 12.18.1. This should be fine, but we should be careful when we update node versions or change this image that this may not be what we expect.

Reply:
Ah yeah, when I copied the circle setup from other node projects running the python apps failed so I just used this as a way to get both node and python in the image